assets/js/filter.js

```diff
@@ -0,0 +1,196 @@
+$(function() {
+    "use strict";
+
+    $('.filtered-table').aplusTableFilter();
+});
+
+/**
+ * Table row filter.
+ */
+(function($, window, document, undefined) {
+    "use strict";
+
+    var pluginName = "aplusTableFilter";
+    var defaults = {};
+    var translationsReady = false;
```

Comment:
Hmm... Maybe there should be a global status register. Then events like `aplus:translation-ready` would indicate that global `aplus.translationReady` variable has changed.

Reply:
Sounds good idea for the future. I also ran into this issue with #390 where some translations require me to wait for the aplus:translation-ready event and others do not require it. It does not seem correct that some translations inside a file wait for the event and others do not, because translations become ready while the script is running. I think this could also bugs with translations in the future.

Reply:
It would be amazing if we had implementation for `_`, which returns lazy object. When ever that object is converted to a string, it would request the translation. Then only render actions would need to wait for translations to become ready,

Reply:
p.s. jQuery ready implements basically the same as here: https://github.com/jquery/jquery/blob/master/src/core/ready.js#L69

Reply:
I'm pretty sure we need to implement these in A+ JS API, like `aPlus.after('translations-ready', self.init);`. TODO: aplus js api :)